Makefile

```diff
@@ -2,47 +2,53 @@
 
 export PATH := $(abspath bin/):${PATH}
 
-# Dependency versions
-KIND_VERSION = 0.20.0
-HELM_DOCS_VERSION = 1.11.0
+##@ General
+
+# Targets commented with ## will be visible in "make help" info.
+# Comments marked with ##@ will be used as categories for a group of targets.
+
+.PHONY: help
+default: help
+help: ## Display this help
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+##@ Development
 
 .PHONY: up
 up: ## Start development environment
 	kind create cluster
 
-.PHONY: stop
-stop: ## Stop development environment
-	# TODO: consider using k3d instead
-	kind delete cluster
-
 .PHONY: down
 down: ## Destroy development environment
 	kind delete cluster
 
-.PHONY: build
-build: ## Build binary
-	@mkdir -p build
-	go build -race -o build/manager ./cmd/manager
+##@ Build
 
 .PHONY: artifacts
-artifacts: helm-chart
+artifacts: generate helm-chart
 artifacts: ## Build artifacts
 
 .PHONY: helm-chart
 helm-chart: ## Build Helm chart
 	@mkdir -p build
 	helm package -d build/ vault
 
+.PHONY: generate
+generate: ## Generate Helm chart documentation
+	helm-docs -s file -c ./vault -t README.md.gotmpl
```

Comment:
I think this won't use the binary downloaded by `make deps`, could you make it so that if it is not in my path, look for it in the bin directory?

Reply:
Should work now